azure-pipelines.yaml

```diff
@@ -35,7 +35,7 @@ steps:
   displayName: set container tag if release
   # update helm charts tag and version if files ready for publishing chart
 - script: |
-    sed -i -e 's/image: mine-support/image: mine-support:$(containerTag)/' ./helm/values.yaml
+    sed -i -e 's/image: ffc-demo/image: ffc-demo$(containerTag)/' ./helm/values.yaml

```

Comment:
I noticed when changing the other service that this line was wrong in the master branch. It would only be used in the helm publish, but worth fixing as the line is changing
It should be:
`sed -i -e 's/image: ffc-demo-calculation-service/image:ffc-demo-calculation-service:$(containerTag)/' ./helm/values.yaml`

be careful to include the `:`!

Reply:
Fixed!